### ISSUE TITLE
500 Server Error when accessing WebDriver documentation

### ISSUE THREAD

**Issue by p0deje** [SeleniumHQ/www.seleniumhq.org#155]
Following http://www.seleniumhq.org/docs/03_webdriver.jsp gives an error:

```
Error: Server Error

The server encountered an error and could not complete your request.
Please try again in 30 seconds.
```

**Comment by juangj** [comment]
(https://travis-ci.org/SeleniumHQ/www.seleniumhq.org/builds/248987501 is the log of the last Travis deployment)

**Comment by p0deje** [comment]
I don't know how website works and can't find anything suspicious in logs :(

**Comment by juangj** [comment]
Same, can't say I really know anything about how the website works. I wasn't able to reproduce the problem in the devserver, either (which makes sense -- my devserver would be built in the same environment as the one I deployed to App Engine).

**Comment by juangj** [comment]
Should be fixed by 1f419bf7cafe0c3893d699d6b87802608ac6582c, though that doesn't explain why I couldn't reproduce locally.
